recipes/_configure_indexes.rb

```diff
@@ -39,9 +39,11 @@
   daily_mb = hash.delete('_maxDailyDataSizeMB')
   padding = hash.delete('_dataSizePaddingPercent')
   default_config = config.fetch('default', {})
+
+  s2_enabled_index =  hash.key?('remotePath') || !default_config['remotePath'].nil?
```

Comment:
minor, we're doing the same check in two different ways here. To make the code a bit easier to read, please update them to be the same.  Either use .key? on both or check for ! .nil? on both.

Reply:
updated to check for remotePath [43ed5fe](https://github.com/cerner/cerner_splunk/pull/241/commits/43ed5fe18075cff99d99acbdcc302e2427146dbb#diff-73c916ec93eae5658af563fbf3f9eedd6ad279c110b632537704da1014ce0c1dR43)